native/android/gradle.properties

```diff
@@ -37,4 +37,6 @@ newArchEnabled=false
 
 # Use this property to enable or disable the Hermes JS engine.
 # If set to false, you will be using JSC instead.
-hermesEnabled=true
\ No newline at end of file
+hermesEnabled=true
+
+org.gradle.parallel=false
```

Comment:
should this really be false?
Due to your description parallel builds are enabled with this pr.
Then its not very intuitive to enable sth by setting it false. This just confuses me

Reply:
You are right, I messed up, will set it to true.